packages/bitcore-wallet-client/lib/api.js

```diff
@@ -293,126 +272,28 @@ API.prototype.validateKeyDerivation = function(opts, cb) {
     _deviceValidated = true;
   }
 
-  var liveOk = (c.canSign() && !c.isPrivKeyEncrypted()) ? testLiveKeys() : true;
-
-  self.keyDerivationOk = hardcodedOk && liveOk;
+  // TODO
+//  var liveOk = (c.canSign() && !c.isPrivKeyEncrypted()) ? testLiveKeys() : true;
+  self.keyDerivationOk = hardcodedOk; // && liveOk;
 
   return cb(null, self.keyDerivationOk);
 };
 
-/**
- * Seed from random with mnemonic
- *
- * @param {Object} opts
- * @param {String} opts.coin - default 'btc'
- * @param {String} opts.network - default 'livenet'
- * @param {String} opts.passphrase
- * @param {Number} opts.language - default 'en'
- * @param {Number} opts.account - default 0
- */
-API.prototype.seedFromRandomWithMnemonic = function(opts) {
```

Comment:
Is there a reason why not check this: `$.checkState(false, 'not supported');` instead remove the function?
Copay uses `seedFromRandomWithMnemonic`.

Reply:
didnt want to remove the code just yet. will remove it soon.

Reply:
removed.